### PR TITLE
Use absolute urls for the node_modules folder to improve module resolution for Peril

### PR DESCRIPTION
**Pull request by orta**
I'm seeing issues with parsing typescript files in Peril. Think this should get it.

### CODE REVIEW

**Comment by orta** [comment]
Interesting

File: source/runner/DangerfileRunner.ts [hunk 1]
```diff
@@ -66,7 +67,7 @@ export async function dangerJestConfig() {
       defaultPlatform: "danger-js",
     },
     moduleNameMapper: [],
-    moduleDirectories: ["node_modules"],
+    moduleDirectories: [path.join(process.cwd(), "node_modules")],
```
Comment (macklinu) [inline source/runner/DangerfileRunner.ts]: I wonder if [`path.resolve()`](https://nodejs.org/api/path.html#path_path_resolve_paths) is more what we're looking for here. From the docs:

> [it] resolves a sequence of paths or path segments into an absolute path.

Comment (orta) [inline source/runner/DangerfileRunner.ts]: Let's give it a shot 👍

Comment (orta) [inline source/runner/DangerfileRunner.ts]: Looks like it doesn't - at least the results are actually the problem that I was having on Peril.

**Comment by orta** [comment]
For myself, errors look like this:

```
  ● with fixtures › can handle a plugin (which is already used in Danger)
    Cannot find module 'danger-plugin-yarn' from '__DangerfilePlugin.js'
      
      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:172:17)
      at Object.<anonymous> (source/runner/_tests/fixtures/__DangerfilePlugin.js:1:65)
      at source/runner/file:/C:/projects/danger-js/source/runner/DangerfileRunner.ts:97:13
      at ensureCleanDangerfile (source/runner/file:/C:/projects/danger-js/source/runner/DangerfileRunner.ts:136:3)
      at Object.<anonymous> (source/runner/file:/C:/projects/danger-js/source/runner/DangerfileRunner.ts:96:3)
      at step (source/runner/DangerfileRunner.ts:32:23)
      at Object.next (source/runner/DangerfileRunner.ts:13:53)
      at source/runner/DangerfileRunner.ts:7:71
      at Object.<anonymous>.__awaiter (source/runner/DangerfileRunner.ts:3:12)
      at Object.runDangerfileEnvironment (source/runner/DangerfileRunner.ts:126:12)
      at Object.<anonymous> (source/runner/_tests/file:/C:/projects/danger-js/source/runner/_tests/_danger_runner.test.ts:153:27)
      at step (source/runner/_tests/_danger_runner.test.ts:32:23)
      at Object.next (source/runner/_tests/_danger_runner.test.ts:13:53)
      at fulfilled (source/runner/_tests/_danger_runner.test.ts:4:58)
  √ creates a working jest config (15ms)
```

Effectively the dependency can't be resolved from inside the eval context

**Comment by orta** [comment]
This isn't used in the post-jest, vm2 world

**Pull request closed without merging**